lib/graphql_practice/metric.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GraphqlPractice.Metric do
+  use Agent
+
+  def start_link([]) do
+    Agent.start_link(fn -> %{} end, name: __MODULE__)
+  end
+
+  def get_count(request) do
+    Agent.get(__MODULE__, fn state ->
+      Map.get(state, request)
```

Comment:
Return 0 instead of nil and then you will always be able to return an integer from this function.

3rd argument for Map.get/3 is the default value if the key is not found.